src/course-outline/configure-modal/BasicTab.jsx

```diff
@@ -1,43 +1,102 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Stack } from '@edx/paragon';
+import { Stack, Form } from '@edx/paragon';
 import { FormattedMessage, injectIntl, useIntl } from '@edx/frontend-platform/i18n';
 import messages from './messages';
 import { DatepickerControl, DATEPICKER_TYPES } from '../../generic/datepicker-control';
 
-const BasicTab = ({ releaseDate, setReleaseDate }) => {
+const BasicTab = ({
+  releaseDate,
+  setReleaseDate,
+  isSubsection,
+  graderType,
+  courseGraders,
+  setGraderType,
+  dueDate,
+  setDueDate,
+}) => {
   const intl = useIntl();
-  const onChange = (value) => {
+  const onChangeReleaseDate = (value) => {
     setReleaseDate(value);
   };
 
+  const onChangeGraderType = (e) => setGraderType(e.target.value);
+
+  const onChangeDueDate = (value) => {
+    setDueDate(value);
+  };
+
+  const createOptions = () => courseGraders.map((option) => (
+    <option value={option}> {option} </option>
+  ));
+
   return (
     <>
-      <h3 className="mt-3"><FormattedMessage {...messages.releaseDateAndTime} /></h3>
+      <h5 className="mt-4 text-gray-700"><FormattedMessage {...messages.releaseDateAndTime} /></h5>
       <hr />
-      <Stack direction="horizontal" gap={5}>
+      <Stack className="mt-3" direction="horizontal" gap={5}>
         <DatepickerControl
           type={DATEPICKER_TYPES.date}
           value={releaseDate}
           label={intl.formatMessage(messages.releaseDate)}
           controlName="state-date"
-          onChange={(date) => onChange(date)}
+          onChange={(date) => onChangeReleaseDate(date)}
         />
         <DatepickerControl
           type={DATEPICKER_TYPES.time}
           value={releaseDate}
           label={intl.formatMessage(messages.releaseTimeUTC)}
           controlName="start-time"
-          onChange={(date) => onChange(date)}
+          onChange={(date) => onChangeReleaseDate(date)}
         />
       </Stack>
+      {
+        isSubsection && (
+          <div>
+            <h5 className="mt-4 text-gray-700"><FormattedMessage {...messages.grading} /></h5>
+            <hr />
+            <Form.Label><FormattedMessage {...messages.gradeAs} /></Form.Label>
+            <Form.Control
+              as="select"
+              defaultValue={graderType}
+              onChange={(value) => onChangeGraderType(value)}
+              data-testid="grader-type-select"
+            >
+              <option value="Not Graded"> Not Graded </option>
+              {createOptions()}
+            </Form.Control>
+            <Stack className="mt-3" direction="horizontal" gap={5}>
+              <DatepickerControl
+                type={DATEPICKER_TYPES.date}
+                value={dueDate}
+                label={intl.formatMessage(messages.dueDate)}
+                controlName="state-date"
+                onChange={(date) => onChangeDueDate(date)}
```

Comment:
Same here, no need to create an arrow function or wrapper func: `onChangeDueDate`.
```suggestion
                onChange={setDueDate}
```